src/ui/download-repo-dialog.cpp

```diff
@@ -18,33 +19,19 @@
 #include "api/server-repo.h"
 #include "download-repo-dialog.h"
 
-namespace {
-
-QString buildMoreInfo(ServerRepo& repo)
-{
-    json_t *object = NULL;
-    char *info = NULL;
-
-    object = json_object();
-    json_object_set_new(object, "is_readonly", json_integer(repo.readonly));
-
-    info = json_dumps(object, 0);
-    QString ret = QString::fromUtf8(info);
-    json_decref (object);
-    free (info);
-    return ret;
+static bool inline isNonEmptyDirectory(const QString &dirpath) {
```

Comment:
The functions/variables in anonymous namespace are identical to the static ones while I saw no encourage of this in [google code style](http://google-styleguide.googlecode.com/svn/trunk/cppguide.html#Namespaces). In fact, I prefer to use anonymous namespace for class definition while the functions and variables are declared static, which makes code more clear that local things are local.